action.yml

```diff
@@ -79,8 +86,9 @@ inputs:
       Note that it is generally bad practice to modify your branch
       in a CI Pipeline. However there are exceptions where this might
       be needed. (use with care).
-
-outputs: {}
+outputs:
+  kubernetesPVC:
+    description: 'The PVC the Kubernetes build has been deployed to'
```

Comment:
Generally I think for newer developers it's easier to understand if at least in the description we describe the acronym; i.e. `The Persistent Volume Claim (PVC) where Kubernetes has put the build artifacts`.

Also, I think semantically we're talking about a PV, not a PVC here.

While we're at it; I would recommend calling this output just `volume` or `artifactVolume` if it's every going to be something different than a `persistent` volume, we might add another output variable called `volumeType`.

So my suggestion would be something like:

```yaml
outputs:
  volume:
    description: 'The Persistent Volume (PV) where the build artifacts have been stored by Kubernetes'
```